### ISSUE TITLE
<< m lock >> command not working.

### ISSUE THREAD

**Issue by abhi18av** [rgcr/m-cli#45]
Hey, here's the output from the terminal

```bash

  File "/usr/local/m-cli/plugins/lock", line 10
    m lock     # lock session"""
                               ^
SyntaxError: Missing parentheses in call to 'print'

```

**Comment by rgcr** [comment]
The latest commit fixes this 59049b3

**Comment by abhi18av** [comment]
Right, thanks for the hard work. It is really useful!
